tap_teamwork/client.py

```diff
@@ -47,7 +43,7 @@ class Client:
     def __init__(self, config: Mapping[str, Any]) -> None:
         self.config = config
         self._session = session()
-        self.base_url = "https://api.teamwork.com/"
+        self.base_url = config.get("base_url", "https://qlik6.teamwork.com/") # from api to qlik6
```

Comment:
Why org specific base url is configured here? if this is supposed to be subdomain specific then we need to read that property from tap-config.